### PR TITLE
CHAIN-292: adjust fee rates in fixtures

### PR DESCRIPTION
**Pull request by bholzman**

### CODE REVIEW

**Review by antonyshchenko — APPROVED** [review]

File: src/main/kotlin/co/chainring/tasks/fixtures/Fixtures.kt [hunk 1]
```diff
@@ -47,7 +47,7 @@ data class Fixtures(
 fun String.toChainSymbol(chainIndex: Int) = "$this${if (chainIndex == 0) "" else "${chainIndex + 1}"}"
 
 fun getFixtures(chainringChainClients: List<BlockchainClient>) = Fixtures(
-    feeRates = FeeRates.fromPercents(maker = 1.0, taker = 2.0),
+    feeRates = FeeRates.fromPercents(maker = 0.01, taker = 0.02),
```
Comment (antonyshchenko) [inline src/main/kotlin/co/chainring/tasks/fixtures/Fixtures.kt:50]: That going to be `0.01%` and `0.02%`

**Comment by bholzman** [comment]
Yup. That’s in-line with what vertex charges.On Jun 10, 2024, at 3:46 PM, Anton Onyshchenko ***@***.***> wrote:﻿
@antonyshchenko commented on this pull request.



In src/main/kotlin/co/chainring/tasks/fixtures/Fixtures.kt:
> @@ -47,7 +47,7 @@ data class Fixtures(
 fun String.toChainSymbol(chainIndex: Int) = "$this${if (chainIndex == 0) "" else "${chainIndex + 1}"}"
 
 fun getFixtures(chainringChainClients: List<BlockchainClient>) = Fixtures(
-    feeRates = FeeRates.fromPercents(maker = 1.0, taker = 2.0),
+    feeRates = FeeRates.fromPercents(maker = 0.01, taker = 0.02),

That going to be 0.01% and 0.02%

File: src/main/kotlin/co/chainring/tasks/fixtures/Fixtures.kt (hunk 1 above)
Comment (imykolenko) [reply src/main/kotlin/co/chainring/tasks/fixtures/Fixtures.kt:50]: that's what on demo right now

Comment (bholzman) [reply src/main/kotlin/co/chainring/tasks/fixtures/Fixtures.kt]: There were a bunch of tests which had hard-coded numbers based on the fee rate of 1% and 2%, and it was going to take a long time to fix them. So instead I am leaving the fee rate at 1% and 2% for tests, but overriding it in demo.

**Review by antonyshchenko — APPROVED** [review]

**Pull request merged**